tests/std/tests/VSO_0000000_instantiate_iterators_misc/test.compile.pass.cpp

```diff
@@ -961,8 +961,10 @@ void memory_test() {
     default_delete<int[]> dd1{default_delete<int[]>{}};
     dd1(new int[5]);
 
+#if _HAS_GARBAGE_COLLECTION_SUPPORT_DELETED_IN_CXX23

```

Comment:
Same, I would have expected those to be there if they have not been deleted

Reply:
Same as https://github.com/microsoft/STL/blob/main/stl/inc/yvals_core.h#L1113-L1117